### PR TITLE
apps wc: Fix fluentd configuration

### LINKED ISSUE #60: It should be possible to set and override the tolerations and nodeSelector for fluentd in wc

### PR DESCRIPTION
**Pull request by OlleLarsson**
**What this PR does / why we need it**:
Removes leftover nodeSelector and tolerations in the values file. Now
everything is only taken from the configuration files.

**Which issue this PR fixes**:
fixes #60 

**Special notes for reviewer**:

**Checklist:**

- [x] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [x] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [x] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

File: WIP-CHANGELOG.md [hunk 1]
```diff
@@ -29,3 +29,4 @@
 - Kibana OIDC logout not redirecting correctly.
 - Getting stuck at selecting tenant when logging in to Kibana.
 - The user fluentd configuration uses its dedicated values for tolerations, affinity and nodeselector.
+- The wc fluentd tolerations and nodeSelector configuration options are now only sepcified in the configuration file.
```
Comment (danielharr) [inline WIP-CHANGELOG.md:32]:
```suggestion
- The wc fluentd tolerations and nodeSelector configuration options are now only specified in the configuration file.
```

**Review by danielharr — CHANGES_REQUESTED** [review]
Nice! Just a small typo in the changelog, otherwise LGTM :slightly_smiling_face:

**Review by danielharr — APPROVED** [review]
LGTM! :slightly_smiling_face:

**Review by Xartos — APPROVED** [review]
LGTM!

**Review by viktor-f — APPROVED** [review]

**Pull request merged**